R/bs-theme.R

```diff
@@ -253,35 +255,43 @@ bootstrap_layer <- function(version) {
       )
     )
 
-    return(sass_layer_merge(bs4_layer, bs4_additions))
+    return(sass_bundle(
+      bs4_bundle,
+      !!bs_sass_bundle_version("bootstrap", version, subname = "additions") := bs4_additions
+    ))
   }
 
   if (version %in% "3") {
+
+    glyphicon <- sass_layer(
+      defaults = list("icon-font-path" = "'glyphicon-fonts/'"),
+      file_attachments = c(
+        "glyphicon-fonts" = lib_file("bootstrap-sass", "assets", "fonts", "bootstrap")
+      )
+    )
+
     # Should match https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss
-    bs3_core <- sass_layer(
-      defaults = list(
-        list("icon-font-path" = "'glyphicon-fonts/'"),
-        bs_sass_files("variables", version = 3)
-      ),
-      declarations = bs_sass_files("mixins", version = 3),
-      rules = bs_sass_files(c(
+    bs3_core <- bs_sass_file_bundle(
+      name = "bootstrap",
+      version = "3",
+      file_version = "3",
+      defaults = "variables",
+      declarations = "mixins",
+      rules = c(
         "normalize", "print", "glyphicons", "scaffolding", "type", "code", "grid",
         "tables", "forms", "buttons", "component-animations", "dropdowns", "button-groups",
         "input-groups", "navs", "navbar", "breadcrumbs", "pagination", "pager", "labels",
         "badges", "jumbotron", "thumbnails", "alerts", "progress-bars", "media",
         "list-group", "panels", "responsive-embed", "wells", "close", "modals",
         "tooltip", "popovers", "carousel", "utilities", "responsive-utilities"
-      ), version = 3),
-      file_attachments = c(
-        "glyphicon-fonts" = lib_file("bootstrap-sass", "assets", "fonts", "bootstrap")
-      ),
-      # Tag this layer so we know we can query the theme_version()
-      tags = "bootstraplib_version_3"
+      )
     )
 
-    return(
-      sass_layer_merge(bs3_core, bs3_accessibility_layer())
-    )
+    return(sass_bundle(
+      bs3_core,
+      !!bs_sass_bundle_version("bootstrap", version, subname = "accessiblity") := bs3_accessibility_bundle(),
+      !!bs_sass_bundle_version("bootstrap", version, subname = "glyphicon") := glyphicon
```

Comment:
I went back-and-forth on whether to even name this bundle at all (since it just let's the CSS rules know where to find font files), but I suppose it might be useful to avoid the overhead of copying over files

```suggestion
      !!bs_sass_bundle_version("bootstrap", version, subname = "glyphicon_fonts") := glyphicon
```